pepper-apis/docs/specification/package.json

```diff
@@ -13,6 +13,7 @@
     "start": "\"$(npm bin)/redoc-cli\" serve --watch src/pepper.yml",
     "lint": "npm run generate-bundle && \"$(npm bin)/speccy\" lint --rules=\"rules.yml\" build/pepper.yml",
     "generate-bundle": "\"$(npm bin)/swagger-cli\" bundle --outfile build/pepper.yml --type yaml src/pepper.yml",
+    "generate-json-bundle": "\"$(npm bin)/swagger-cli\" bundle --outfile build/pepper.json src/pepper.yml",
     "generate-docs": "\"$(npm bin)/redoc-cli\" bundle --output build/pepper.html src/pepper.yml",
```

Comment:
Added the json bundle generation to package.json as well (though we are running these builds from a bash script).